lib/multi.c

```diff
@@ -1392,9 +1392,10 @@ static CURLMcode multi_runsingle(struct Curl_multi *multi,
       }
 
       if(!result) {
-        if(async)
+        if(async) {
           /* We're now waiting for an asynchronous name lookup */
           multistate(data, CURLM_STATE_WAITRESOLVE);
+        }
```

Comment:
this looks like just (unrelated) white space changes, avoid those!

Reply:
done.